src/lib/db.js

```diff
@@ -26,7 +26,8 @@ function executeSQLPlus (sqlFilepath) {
     const host = connection.host
     const port = connection.port
     const database = connection.database
-    const login = user + '/' + password + '@' + host + ':' + port
+    // TODO: Oracle connection string passed as configuration to ledger
+    const login = user + '/' + password + '@' + host + '/xe:' + port
```

Comment:
Yeah, but the amazing thing is that actually solved the connection issue. I found another way to do that so this change will be undone in next commit.